frontend/components/maps/MapHousing.js

```diff
@@ -0,0 +1,316 @@
+import React from "react";
+import PropTypes from "prop-types";
```

Comment:
One thing that would be worth doing especially if you're finding it hard to traverse this file @erinliu1 @pauladc is to refactor out the components to separate files. It's certainly hefty!